platforms/defaults_windows.go

```diff
@@ -53,6 +54,31 @@ func (m windowsmatcher) Match(p specs.Platform) bool {
 		if strings.HasPrefix(p.OSVersion, m.osVersionPrefix) {
 			return true
 		}
+		if m.isClientOS && p.OSVersion != "" {
+			split := strings.Split(p.OSVersion, ".")
+			if len(split) >= 3 {
+				major, err := strconv.Atoi(split[0])
+				if err != nil {
+					return false
+				}
+				minor, err := strconv.Atoi(split[1])
+				if err != nil {
+					return false
+				}
+				build, err := strconv.Atoi(split[2])
+				if err != nil {
+					return false
+				}
+
+				if (major == 10 && minor == 0 && build >= 20348) ||
```

Comment:
not: Should be able to use the server 2022 constant from hcsshim: https://github.com/microsoft/hcsshim/blob/main/osversion/windowsbuilds.go#L72